### DISCUSSION TITLE
Is the "Update every" setting the same as a crontab or does it just update when the app is kept open?

### DISCUSSION THREAD

**Discussion by SinTan1729** [cboxdoerfer/fsearch#480] [Q&A] [answered]
![image](https://github.com/cboxdoerfer/fsearch/assets/35728385/d55ffd3c-b5cc-45b7-8eb6-94195562b964)

**Comment by cboxdoerfer** [comment, marked as answer]
They're independent and the GUI setting only applies while the app is running.

**Reply by SinTan1729** [reply]
That makes it clear. Thanks.
